alerts.css

```diff
@@ -0,0 +1,66 @@
+.alert-container{
+  border-radius: 0.48rem;
+  padding: 1.6rem 4.8rem 1.6rem 1.6rem;
+  font-size: 1.6rem;
+  margin: 1.6rem 0;
+  position: relative;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.primary-alert{
+  color: #004085;
+  background-color: #cce5ff;
+  border-color: #b8daff;
+}
+
+.success-alert{
+  color: #155724;
+  background-color: #d4edda;
+  border-color: #c3e6cb;
+}
+
+.secondary-alert{
+  color: #383d41;
+  background-color: #e2e3e5;
+  border-color: #d6d8db;
+}
+
+.danger-alert{
+  color: #721c24;
+  background-color: #f8d7da;
+  border-color: #f5c6cb;
+}
+
+.warning-alert{
+  color: #856404;
+  background-color: #fff3cd;
+  border-color: #ffeeba;
+}
```

Comment:
as you are using color again again , you should use color with variable name . it will improve code readability